plan/gen_physical_plans.go

```diff
@@ -245,12 +255,11 @@ func (p *LogicalJoin) constructIndexJoin(prop *requiredProp, innerJoinKeys, oute
 
 // getIndexJoinByOuterIdx will generate index join by outerIndex. OuterIdx points out the outer child,
```

Comment:
update this comment to describe all the behavior of this function

Reply:
ok